packages/keystone/fields/Select/implementation.js

```diff
@@ -20,7 +22,13 @@ class Select extends KSSelect.implementation {
         if (this.graphQLReturnType) {
             return []
         } else {
-            return super.getGqlAuxTypes()
+            return [
```

Comment:
I guess something wrong with alignment here?
<img width="583" alt="image" src="https://github.com/open-condo-software/condo/assets/50069872/ff3e340b-37f6-480e-b467-c3086d3e80ae">

Reply:
It's possible that this doesn't affect anything, but just in case I'll leave it as it was done in the original implementation)